subhub/customer.py

```diff
@@ -73,6 +78,10 @@ def existing_or_new_customer(
     origin_system: str,
     display_name: str,
 ) -> Customer:
+    if not origin_system.lower() in CFG.ALLOWED_ORIGIN_SYSTEMS:
```

Comment:
with the `.lower()` now we have these _uglier_ if conditionals.  What requirement is the lower() meeting?

Reply:
I was taking case sensitivity out of the mix for callers and also application configuration for deployment agents (us).